Project/Engine/src/sysFrame/System/Math/MathUtils.cpp

```
@@ -136,3 +136,16 @@
     std::uniform_real_distribution<float> dist(min, max);
     return dist(random);
 }
+
+float MathUtils::Distance(const Vector3& a, const Vector3& b) {
```

cppcheck (2.10-2)

[style] The function 'Distance' is never used.

(unusedFunction)